vanilla-javascript-demo/package.json

```diff
@@ -8,7 +8,7 @@
     "start": "parcel index.html --open"
   },
   "dependencies": {
-    "@pki-chemistry/chemdraw-js": "22.3.0-develop-4580"
```

Comment:
Yes, this won't work - the whole point of this is that the user doesn't have access to Artifactory.